Modules/SlashCommands.cs

```diff
@@ -235,6 +236,14 @@ public class SlashCommands : ApplicationCommandModule
                 await ctx.RespondAsync($"{Program.cfgjson.Emoji.Error} The source user has no warnings to transfer.", Warnings.GenerateWarningsEmbed(sourceUser));
                 return;
             }
+            else if (merge)
+            {
+                await Program.db.KeyDeleteAsync(sourceUser.Id.ToString());
+                foreach (var warning in sourceWarnings)
+                {
+                    await Program.db.HashSetAsync(targetUser.Id.ToString(), warning.Name, warning.Value);
+                }
+            }
             else if (targetWarnings.Length > 0 && !forceOverride)
             {
                 await ctx.RespondAsync($"{Program.cfgjson.Emoji.Warning} **CAUTION**: The target user has warnings.\n\n" +
```

Comment:
The warning output when trying to overwite warnings should mention the existence of merge as well.

Also since it won't let me separately comment on lines too far down, the success message should also mention whether or not a merge was performanced (You can click the slash command, but it isn't immediately obvious to the other mods what was just done)

Reply:
I moved the `KeyDeleteAsync` to after the `foreach` loop now.

Reply:
I think you replied to the wrong conversation?

Reply:
My idea is to use command buttons if the target user has warnings and the moderator did not specify whether to merge or override.

Reply:
I think that a button would make it too easy to make a mistake. These options are destructive operations that shouldn't be taken lightly

Reply:
Alright.

Reply:
Do you think this would be fine?

> If you are sure you want to **OVERRIDE** and **DELETE** these warnings, please consider the consequences before adding `force_override: true` to the command.
If you wish to **NOT** override the target's warnings, please use `merge: true` instead

Reply:
Sounds good to me, though make sure you add a `.` on the end ;)

Reply:
:)

Reply:
I modified it so if mods did not specify the operation, it would notify about both override and merge.

Reply:
Thank you!
Do you think you could add a small thing to the success message specifying the transfer that happened was a merge? Should be all done after that.

Reply:
Alright.

Reply:
I made a commit for that. What do you think?

Reply:
Oh it failed checks LOL.

Reply:
Yeah build errors, check them out down from here https://github.com/Erisa/Cliptok/runs/4074558064?check_suite_focus=true#step:5:199

Reply:
Error resolved now.